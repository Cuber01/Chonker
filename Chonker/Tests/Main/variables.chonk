var number a = 1;
var string b = "hello", number c = 2;

b = "not hello"

print a;
print b;
print c;