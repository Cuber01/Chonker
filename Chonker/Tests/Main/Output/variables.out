1
not hello
2